ecosystem.config.js

```diff
@@ -87,8 +87,8 @@ module.exports = {
         WRITE_STORAGE_TYPE : "datatransfer.storage.FolderStorage",
         READ_STORAGE_TYPE : "datatransfer.storage.S3Storage",
         LOG_FILE_NAME : "s3-landing-to-local.log",
-        PROCESS_INTERVAL : 4,
-        MAX_FILES_BATCH : 25,
+        PROCESS_INTERVAL : 300,
```

Comment:
Can this not be a lower number so it is more frequent or is it controlled by a cronjob or pm?

Reply:
Reflects Prod.